### PR TITLE
Add gatsby website

### PR DESCRIPTION
**Pull request by ibgreen**

### CODE REVIEW

**Comment by Pessimistress** [comment]
Can we just delete the old website?

File: website/html.config.js [hunk 1]
```diff
@@ -20,11 +20,16 @@
 
 module.exports = {
 
-  title: 'probe.gl',
```
Comment (Pessimistress) [inline website/html.config.js]: Copy paste?

File: website/src/config.js [hunk 2]
```diff
@@ -19,12 +19,10 @@
 // THE SOFTWARE.
 
 export const PROJECT_TYPE = 'github';
-
-export const PROJECT_NAME = 'probe.gl';
```
Comment (Pessimistress) [inline website/src/config.js]: Ditto

**Comment by ibgreen** [comment]
> Can we just delete the old website?

Unfortunately while things work well in develop mode there is still a style related error during static page generation.

I'll land this so that we can make a localized fix, and we'll remove the other website when this is confirmed to be working.

**Comment by ibgreen** [comment]
> Copy paste?

Hmm, yes, looks like math.gl has some residual code in its website directory which made its way over here during directory copy, I'll put up a PR deleting that later.

**Pull request merged**